dev-env/lib/dade-common

```diff
@@ -127,6 +127,8 @@ execTool() {
   local outputName=$2
   local binary=$3
   shift 3
+  [[ $PATH =~ (^|.*:)$DADE_DEVENV_DIR/bin($|:.*) ]] || \
```

Comment:
I'm not sure if this is the best place to add this, but it seems to get the job done.